lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -721,9 +721,22 @@ sub cluster_jobs {
         @_
     );
 
-    my $jobs = $args{jobs};
-    return $jobs if defined $jobs->{$self->id};
-    $jobs->{$self->id} = {
+    my $jobs          = $args{jobs};
+    my $job_id        = $self->id;
+    my $job           = $jobs->{$job_id};
+    my $skip_children = !!$args{skip_children};
```

Comment:
why use `!!` here?  we only want to get `1` ?

Reply:
Right, I removed it.